pkgs/tools/networking/wireguard-tools/default.nix

```diff
@@ -47,7 +47,12 @@ stdenv.mkDerivation rec {
     done
   '';
 
-  passthru.updateScript = ./update.sh;
+  passthru = {
+    updateScript = ./update.sh;
+    tests = {
+      inherit (nixosTests) wg-quick wireguard-generated wireguard-namespaces;
```

Comment:
I think we should also have plain `wireguard` here (the file at `nixos/tests/wireguard/default.nix` is also a test).

Reply:
Fixed this.